diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/comments/HeaderCommentRule.kt

```diff
@@ -185,7 +185,7 @@ class HeaderCommentRule(private val configRules: List<RulesConfig>) : Rule("head
 
         if (copyrightWithCorrectYear.isNotEmpty()) {
             WRONG_COPYRIGHT_YEAR.warnAndFix(configRules, emitWarn, isFixMode, "year should be $curYear", node.startOffset, node) {
-                (headerComment as LeafElement).replaceWithText(headerComment.text.replace(copyrightText, copyrightWithCorrectYear))
+                (headerComment as? LeafElement)?.replaceWithText(headerComment.text.replace(copyrightText, copyrightWithCorrectYear))
```

Comment:
Why was the exception happening?

Reply:
Because there can be no copyright in file

Reply:
Then why didn't it appear before? Errors started to occur only when the year has changed, all tests have successfully passed with current year being 2020.

Reply:
I still don't get why it's nullable. Is this fix actually being executed somewhere?